Integrations/integration-Pwned.yml

```
@@ -76,12 +77,26 @@ script:
         });
         compSites = compSites.sort();
         var compEmail = {};
+        var dBotScore = 0;
         if (compSites.length > 0) {
+            dBotScore = DEFAULT_DBOT_SCORE;
+            if (DEFAULT_DBOT_SCORE === 3){
+                email.Malicious = {
+                    Vendor: 'Pwned',
+                    Description: 'The email has been compromised',
+                }
+            }
             addCompromised(compEmail, outputPaths.email, {
                 'Address': email,
-                'Compromised': {Vendor:'Pwned', Reason: compSites.join(', ')}
+                'Compromised': {Vendor:'Pwned', Reason: compSites.join(', '), dBotScore: DEFAULT_DBOT_SCORE}
```

anyway, there shouldn't dbot score here 
```suggestion
                'Compromised': {Vendor:'Pwned', Reason: compSites.join(', '), DBotScore: DEFAULT_DBOT_SCORE}
```